modules/joex/src/main/resources/reference.conf

```diff
@@ -648,11 +668,32 @@ Docpell Update Check
     # (where ocr is not necessary). In this case, the pdf will be
     # converted to PDF/A.
     ocrmypdf = {
+    # Custom argument mappings for this program.
+    arg-mappings = {
+      "ocr_lang" = {
+        value = "{{lang}}"
+        # Custom Language Mappings Below
+        # Japanese Vertical Mapping
+        mappings = [
+          {
+            matches = "jpn_vert"
+            args = [ "jpn_vert", "--pdf-renderer", "sandwich", "--tesseract-pagesegmode", "5" ]
+          },
+        # Start Other Custom Language Mappings Here
+        # Default Mapping Below
+          {
+            matches = ".*"
+            args = [ "{{lang}}" ]
+          }
+        ]
+      }
+    }
       enabled = true
       command = {
         program = "ocrmypdf"
+        # Default arguments for all processing go below.
         args = [
-          "-l", "{{lang}}",
+          "-l", "{{ocr_lang}}",
```

Comment:
Only a very nitpicky nitpick :) Would it maybe make sense to also move the `-l` into the mappings? Now it would be required to make the `-l` option work from all mappings, but perhaps some are not even using it (I know very bad example, just want to illustrate that the mapped args need to consider that they are placed exactly after `-l`)